demo/makefile

```diff
@@ -1,5 +1,5 @@
 # Address-sanitizer library
```

Comment:
Can we set these flags on the command line rather than modifying them in the Makefile?